### ISSUE TITLE
promote-config comments out `warn: true` when not snoozed

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-releng-automation#187]
In 8832e34 (and subsequently a27e9dd) we modified the `sed` command to comment out `warn: true` attributes on denylist entries. IIUC this is the right thing to do if a test is snoozed.

```
- pattern: coreos.ignition.ssh.key
  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1553
  snooze: 2024-05-13
  warn: true
  platforms:
    - azure
```

so we want to comment out the snooze (so the test behavior is no longer affected by time, since we are promoting to a production stream) and the warn, because we want to not run the test at all (it's denied, not snoozed). So we end up with after the promote-config script runs:

```
- pattern: coreos.ignition.ssh.key
  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1553
  # snooze: 2024-04-29 (disabled on promotion)
  # warn: true (disabled on promotion)
  platforms:
    - azure
```

In this case the test won't run in the production run.

Now there is another case where we have permanently marked a test as `warn: true`. We did this for https://github.com/coreos/fedora-coreos-tracker/issues/1659 because we don't know when we'll actually be able to spend time to understand the underlying flakiness of the test.

```
- pattern: coreos.boot-mirror*
  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1659
  warn: true
  arches:
    - ppc64le
```

Now, the question is: do we want this `warn: true` to be commented out after promotion:

```
- pattern: coreos.boot-mirror*
  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1659
  # warn: true (disabled on promotion)
  arches:
    - ppc64le
```

I think the answer is no, because we don't want the test to all of a sudden become fatal when it hasn't been fatal in development all the way until this point of promotion.

**Comment by dustymabe** [comment]
TL;DR the behavior right now makes it more likely a production run will fail and need to be retried on a test flake. This might prove frustrating if we are building under a time constraint and have to restart a build.

**Comment by dustymabe** [comment]
ok, actually my understanding is wrong.

>  because we don't want the test to all of a sudden become fatal when it hasn't been fatal in development all the way until this point of promotion.

This isn't what is happening here. Commenting out `warn: true` actually means that the test doesn't get run (it's fully denied). Here's some output from a recent `stable` ppc64le run:

```
[2024-05-07T04:42:37.753Z] ������  Skipping kola test pattern "coreos.boot-mirror*":
```
